requirements.dev.txt

```diff
@@ -8,7 +8,7 @@ asgiref==3.8.1 \
     --hash=sha256:3e1e3ecc849832fe52ccf2cb6686b7a55f82bb1d6aee72a58826471390335e47 \
     --hash=sha256:c343bd80a0bec947a9860adb4c432ffa7db769836c64238fc34bdc3fec84d590
     # via
-    #   -c /home/runner/work/airlock/airlock/requirements.prod.txt
+    #   -c /home/becky/datalab/airlock/requirements.prod.txt
```

Comment:
(not your problem) it's annoying that these urls are being rewritten everytime we do this (it affects me too)

Reply:
Yeah, it's been happening for a while - I think we just decided to ignore it. Although, I think I'm actually going to change these back so that they don't automatically make the update-dependencies action make a PR